pkg/test/runner/config.go

```diff
@@ -85,6 +85,9 @@ type TestCaseConfig struct {
 	// Skip means should this test case be skipped. Default: false
 	Skip bool `json:"skip,omitempty" yaml:"skip,omitempty"`
 
+	// SkipIfNotInCI means should this test case be skipped if the env var KPT_E2E_IN_CI is "false" or not set. Default: false
```

Comment:
"means this test case should be..."

Reply:
Thanks for the catch! Copy paste mistake. Fixed.